behat/symfony2-extension/2.1/features/bootstrap/bootstrap.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require __DIR__.'/../../vendor/autoload.php';
+
+// The check is to ensure we don't use .env in production
+if (!getenv('APP_ENV')) {
```

Comment:
In such example, it would mean that the `APP_ENV` environment variable is set and therefore `new Dotenv` is not called, right?